crypto/fipsmodule/rand/rand.c

```diff
@@ -351,6 +385,15 @@ void RAND_bytes_with_additional_data(uint8_t *out, size_t out_len,
       state = &stack_state;
     }
 
+#if defined(BORINGSSL_FIPS) && defined(JITTER_ENTROPY)
+    // Initialize the thread-local Jitter instance.
+    state->jitter_ec = NULL;
+    state->jitter_ec = jent_entropy_collector_alloc(0, JENT_FORCE_FIPS);
+    if (!state->jitter_ec) {
```

Comment:
Should this be: 
```suggestion
    if (state->jitter_ec == NULL) {
```
?